zerver/lib/actions.py

```diff
@@ -3907,7 +3911,7 @@ def gather_subscriptions_helper(user_profile: UserProfile,
 
         # Important: don't show the subscribers if the stream is invite only
         # and this user isn't on it anymore.
-        if stream["invite_only"] and not sub["active"]:
+        if stream["invite_only"] and (not sub["active"] and not user_profile.is_realm_admin):
```

Comment:
This could be written more clearly as `and not (sub['active'] or user_profile.is_realm_admin)` -- switched when merging.